api/v1/openapi.yaml

```
@@ -1460,7 +1460,9 @@ definitions:
       last-updated:
         description: Time of last health update
         type: string
-
+      count:
+        description: How many times in a row has this status been reported.
```

Nit: the description here is formatted as a question:

> How many times in a row has this status been reported?

Can we reword this to be a statement?

```suggestion
        description: The number of consecutive status updates.
```

Another option:

```The number of times in a row that the status has been reported.```